### ISSUE TITLE
[bug]: describe column not found

### ISSUE THREAD

**Issue by abhinavDhulipala** [danielgafni/dagster-polars#54]
We are currently on `dagster-polars==0.1.5` and we tried to upgrade to 2.2, but are getting the following failures on CI

```bash
polars.exceptions.ColumnNotFoundError: describe
FAILED 
```

### Platform info
We are running in a x86 docker container. 

Polars version:

```bash
polars==0.20.6
```

platform:

```bash
root@50efadb560a1:/src# uname -a
Linux 50efadb560a1 6.2.0-39-generic #40~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Thu Nov 16 10:53:04 UTC 2 x86_64 x86_64 x86_64 GNU/Linux
```

### Details
Here is the full relevant stack trace

```bash
polars.exceptions.ColumnNotFoundError: describe
Stack Trace:
  File "/usr/local/lib/python3.10/dist-packages/dagster/_core/execution/api.py", line 764, in job_execution_iterator
    for event in job_context.executor.execute(job_context, execution_plan):
  File "/usr/local/lib/python3.10/dist-packages/dagster/_core/executor/in_process.py", line 55, in execute
    yield from iter(
  File "/usr/local/lib/python3.10/dist-packages/dagster/_core/execution/api.py", line 875, in __iter__
    yield from self.iterator(
  File "/usr/local/lib/python3.10/dist-packages/dagster/_core/executor/in_process.py", line 26, in inprocess_execution_iterator
    yield from inner_plan_execution_iterator(
  File "/usr/local/lib/python3.10/dist-packages/dagster/_core/execution/plan/execute_plan.py", line 121, in inner_plan_execution_iterator
    for step_event in check.generator(
  File "/usr/local/lib/python3.10/dist-packages/dagster/_core/execution/plan/execute_plan.py", line 359, in dagster_event_sequence_for_step
    raise dagster_user_error.user_exception
  File "/usr/local/lib/python3.10/dist-packages/dagster/_core/execution/plan/utils.py", line 54, in op_execution_error_boundary
    yield
  File "/usr/local/lib/python3.10/dist-packages/dagster/_utils/__init__.py", line 467, in iterate_with_context
    next_output = next(iterator)
  File "/usr/local/lib/python3.10/dist-packages/dagster/_core/execution/plan/execute_step.py", line 738, in _gen_fn
    gen_output = output_manager.handle_output(output_context, output.value)
  File "/usr/local/lib/python3.10/dist-packages/dagster/_core/storage/upath_io_manager.py", line 448, in handle_output
    custom_metadata = self.get_metadata(context=context, obj=obj)
  File "/usr/local/lib/python3.10/dist-packages/dagster_polars/io_managers/base.py", line 317, in get_metadata
    return get_polars_metadata(context, df) if df is not None else {"missing": MetadataValue.bool(True)}
  File "/usr/local/lib/python3.10/dist-packages/dagster_polars/io_managers/utils.py", line 125, in get_polars_metadata
    "stats": MetadataValue.json(get_polars_df_stats(df)),
  File "/usr/local/lib/python3.10/dist-packages/dagster_polars/io_managers/utils.py", line 108, in get_polars_df_stats
    return {
  File "/usr/local/lib/python3.10/dist-packages/dagster_polars/io_managers/utils.py", line 109, in <dictcomp>
    col: {stat: describe[col][i] for i, stat in enumerate(describe["describe"].to_list())}
  File "/usr/local/lib/python3.10/dist-packages/polars/dataframe/frame.py", line 1731, in __getitem__
    return self.get_column(item)
  File "/usr/local/lib/python3.10/dist-packages/polars/dataframe/frame.py", line 7068, in get_column
    return wrap_s(self._df.get_column(name))
The above exception occurred during handling of the following exception:
```

### Notes

Seems like the problem lies [here](https://github.com/danielgafni/dagster-polars/blob/master/dagster_polars/io_managers/utils.py#L109) but looking at the blame, it hasn't changed for 7 months. Is this function invoked some where new? Do we have to have all of our df's implement a `describe` col? If so, where is this documented?

**Comment by danielgafni** [comment]
Hey @abhinavDhulipala!
This is caused by `polars` changing the output column name of `DataFrame.describe` from `describe` to `statistic`. 
I'll submit a fix.

**Comment by danielgafni** [comment]
Fix available in `0.3.0`

**Comment by abhinavDhulipala** [comment]
Awesome thanks guys!
